sdk/core/azure-core-amqp/samples/eventhub_token_writer_sample/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright(c) Microsoft Corporation.All rights reserved.
+# SPDX - License - Identifier : MIT
+
+cmake_minimum_required(VERSION 3.15)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+project(eventhub_sas_writer_sample)
+
+#find_package(azure-identity-cpp CONFIG REQUIRED)
```

Comment:
remove commented out find_package?

Looks like this sample depends on azure-identity.

Reply:
THe SAS token shouldn't need identity.